library/unwind/Cargo.toml

```diff
@@ -24,5 +24,12 @@ cfg-if = "0.1.8"
 cc = "1.0.67"
 
 [features]
+
+# Only applies for Linux and Fuchsia targets
+# Static link to the in-tree build of llvm libunwind
 llvm-libunwind = []
+
+# Only applies for Linux and Fuchsia targets
+# If crt-static is enabled, static link to `libunwind.a` provided by system
+# If crt-static is disabled, dynamic link to `libunwind.so` provided by system
 system-llvm-libunwind = []
```

Comment:
So `llvm-libunwind` and `system-llvm-libunwind` are actually incompatible, what the...
I always assumed that the later "elaborates" for former and tells which exactly libunwind is used if it's indeed used.
Such is life with cargo features.